homeassistant/components/integration/config_flow.py

```diff
@@ -87,6 +88,14 @@
                 translation_key=CONF_UNIT_TIME,
             ),
         ),
+        vol.Optional(CONF_MAX_AGE, default=0): selector.NumberSelector(
+            selector.NumberSelectorConfig(
+                min=0,
+                max=1000000000,
```

Comment:
Why do we have this nonsensical max value, does it not work if it's omitted?

Reply:
Good question. There is no need for this max and I've removed it.